test/MarketMaker.js

```diff
@@ -162,6 +215,69 @@ contract('marketMaker', function(accounts) {
     await help.waitToBlock(startBlock+blocksPerPeriod*numberOfMonth, accounts);
   };
 
+  it("Should return the correct sellPrice on every period", async function() {
+    token = await simulateCrowdsale(100, [40,30,20,10,0], accounts);
+
+    const startBlock = web3.eth.blockNumber+10;
+    const blocksPerPeriod = 5;
+    const initialPriceSpread = 100500;
+
+    mm = await LifMarketMaker.new(
+      token.address, startBlock, blocksPerPeriod, 24,
+      accounts[1], initialPriceSpread,
+      {value: web3.toWei(8, 'ether'), from: accounts[0]}
+    );
+
+    const initialSellPrice = parseInt(await mm.initialSellPrice());
+
+    await mm.calculateDistributionPeriods({from: accounts[4]});
+    await mm.calculateSellPricePeriods({from: accounts[3]});
+
+    help.debug('MM balance:', parseInt( web3.eth.getBalance(mm.address) ));
+    help.debug('Start block', parseInt( await mm.startBlock.call() ));
+    help.debug('Blocks per period', parseInt( await mm.blocksPerPeriod.call() ));
+    help.debug('Foundation address', await mm.foundationAddr.call() );
+    help.debug('Initial Wei', parseInt( await mm.initialWei.call() ));
+    help.debug('Initial Buy Price', parseInt( await mm.initialBuyPrice.call() ));
+    help.debug('Initial Sell Price', parseInt( await mm.initialSellPrice.call() ));
```

Comment:
can we put an assert on the expected `initialSellPrice`? It would be nice to be able to "see" that price here, without having to run the test to see it in the console